extension/src/tdigest.rs

```diff
@@ -655,11 +646,10 @@ mod tests {
             assert_eq!(buffer, expected);
 
             let expected = pgx::varlena::rust_byte_slice_to_bytea(&expected);
-            let new_state =
-                tdigest_deserialize_inner(bytea(pg_sys::Datum::from(expected.as_ptr())));
+            let mut new_state =
+                tdigest_deserialize_inner(bytea(pgx::Datum::from(expected.as_ptr())));
```

Comment:
I think we actually need to go the other way, and change all the `pgx::Datum` to `pg_sys::Datum` for pgx 0.6.

Reply:
Hehe, oops.  Thanks!